admin-dev/themes/new-theme/scss/components/layout/_nav_bar.scss

```
@@ -71,7 +72,7 @@
       display: flex;
       align-items: center;
       padding: 0.875rem;
-      border-bottom: 1px solid $white;
+      border-bottom: 1px solid #bebebe;
```

```suggestion
      border-bottom: 1px solid $gray-dark-text;
```

Please use a variable